src/containerapp/azext_containerapp/_archive_utils.py

```diff
@@ -186,6 +186,10 @@ def _load_dockerignore_file(source_location, original_docker_file_name):
 
 
 def _archive_file_recursively(tar, name, arcname, parent_ignored, parent_matching_rule_index, ignore_check):
+    if os.path.isfile(name) and (arcname == "" or arcname == None):
```

Comment:
```suggestion
    if os.path.isfile(name) and (arcname == "" or arcname is None):
```

Reply:
Hi @daniv-msft  Please fix the CI issue.

Reply:
Thanks for catching